From f8a0371ee0bd31400d126ffb8eca130f52df9098 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:05 -0400
Subject: [fs] nfsd: nfsd4_create_clid_dir prints uninitialized data

Message-id: <1403560314-32525-4-git-send-email-bfields@redhat.com>
Patchwork-id: 84449
O-Subject: [Patch RHEL7 003/172] nfsd4: nfsd4_create_clid_dir prints uninitialized data
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

Take the easy way out and just remove the printk.

Reported-by: David Howells <dhowells@redhat.com>
Upstream: 248f807b479145194a83c5270440b3f51c1836d7
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4recover.c b/fs/nfsd/nfs4recover.c
index 4e9a21d..8462c6c 100644
--- a/fs/nfsd/nfs4recover.c
+++ b/fs/nfsd/nfs4recover.c
@@ -173,8 +173,6 @@ nfsd4_create_clid_dir(struct nfs4_client *clp)
  int status;
  struct nfsd_net *nn = net_generic(clp->net, nfsd_net_id);
 
- dprintk("NFSD: nfsd4_create_clid_dir for \"%s\"\n", dname);
-
  if (test_and_set_bit(NFSD4_CLIENT_STABLE, &clp->cl_flags))
   return;
  if (!nn->rec_file)
-- 
1.7.1